src/app/components/forms/user-details-form/user-details-form.component.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../styles/colors';
+
+.user-details-form-button-container {
+  margin: 40px 0;
+  text-align: center;
+
+  .user-details-form-save-message {
+    color: green;
```

Comment:
Color `green` should be written in hexadecimal form as `#008000`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorkeyword)